src/Root.vue

```diff
@@ -107,7 +107,7 @@
       </span>
       <span class="link">
         <a
-          href="https://docs.sharedstake.org/"
+          href="https://docs.sharedstake.finance/"
```

Comment:
nah lets keep .org. 
the .finance domain is like a lot more per year and i wanna get rid of it eventually.

Reply:
oh nvm. guess i gotta create the dns link first